test/freenet/support/FieldTrimSecondTest.java

```diff
@@ -15,15 +15,15 @@ public class FieldTrimSecondTest extends TestCase {
 	/**
 	 * Correct result in bytes matched by index with input.
 	 */
-	private static final int[] output = { 50 * 1024, 3 * 1024 * 1024 / 2, 128 * 1000, 20 * 1024, 5800 };
+	private static final int[] output = { 50 * 1024, 3 * 1024 * 1024 / 2, (128 / 8) * 1000, 20 * 1024, 5800 };
 
 	public void test() {
 		assert input.length == output.length;
 
 		int parsed;
 		for (int i = 0; i < input.length; i++) {
 			parsed = Fields.parseInt(Fields.trimPerSecond(input[i]));
-			System.out.format("Input: %s\tParsed: %d\tIntended: %d\n", input[i], parsed, output[i]);
+			// System.out.format("Input: %s\tParsed: %d\tIntended: %d\n", input[i], parsed, output[i]);
```

Comment:
Don’t comment it out, just remove it, please…

Reply:
Done — thank you!